src/components/playback/brightnessosd.js

```diff
@@ -1,171 +1,173 @@
-        });
+    dom.removeEventListener(elem, dom.whichTransitionEvent(), onHideComplete, {
+        once: true
+    });
 
-        elem.classList.remove('hide');
+    elem.classList.remove('hide');
 
-        // trigger reflow
-        void elem.offsetWidth;
+    // trigger reflow
+    void elem.offsetWidth;
 
-        requestAnimationFrame(function () {
-            elem.classList.remove('iconOsd-hidden');
+    requestAnimationFrame(function () {
+        elem.classList.remove('iconOsd-hidden');
 
-            hideTimeout = setTimeout(hideOsd, 3000);
-        });
-    }
+        hideTimeout = setTimeout(hideOsd, 3000);
+    });
+}
 
-    function clearHideTimeout() {
-        if (hideTimeout) {
-            clearTimeout(hideTimeout);
-            hideTimeout = null;
-        }
+function clearHideTimeout() {
+    if (hideTimeout) {
+        clearTimeout(hideTimeout);
+        hideTimeout = null;
     }
+}
 
-    function hideOsd() {
+function hideOsd() {
 
-        clearHideTimeout();
+    clearHideTimeout();
 
-        var elem = osdElement;
-        if (elem) {
+    var elem = osdElement;
+    if (elem) {
 
-            if (enableAnimation) {
-                // trigger reflow
-                void elem.offsetWidth;
+        if (enableAnimation) {
+            // trigger reflow
+            void elem.offsetWidth;
 
-                requestAnimationFrame(function () {
-                    elem.classList.add('iconOsd-hidden');
+            requestAnimationFrame(function () {
+                elem.classList.add('iconOsd-hidden');
 
-                    dom.addEventListener(elem, dom.whichTransitionEvent(), onHideComplete, {
-                        once: true
-                    });
+                dom.addEventListener(elem, dom.whichTransitionEvent(), onHideComplete, {
+                    once: true
                 });
-            } else {
-                onHideComplete.call(elem);
-            }
+            });
+        } else {
+            onHideComplete.call(elem);
         }
     }
-
-    function setIcon(iconElement, icon) {
-        iconElement.classList.remove('brightness_high');
-        iconElement.classList.remove('brightness_medium');
-        iconElement.classList.remove('brightness_low');
-        iconElement.classList.add(icon);
-    }
-
-    function updateElementsFromPlayer(brightness) {
-
-        if (iconElement) {
-            if (brightness >= 80) {
-                setIcon(iconElement, 'brightness_high');
-            } else if (brightness >= 20) {
-                setIcon(iconElement, 'brightness_medium');
-            } else {
-                setIcon(iconElement, 'brightness_low');
-            }
-        }
-        if (progressElement) {
-            progressElement.style.width = (brightness || 0) + '%';
+}
+
+function setIcon(iconElement, icon) {
+    iconElement.classList.remove('brightness_high');
+    iconElement.classList.remove('brightness_medium');
+    iconElement.classList.remove('brightness_low');
```

Comment:
```suggestion
    iconElement.classList.remove('brightness_high', 'brightness_medium', 'brightness_low');
```
slightly better looking and minified :)

Reply:
Agree.
FYI, there is similar code elsewhere (#898). All of them can be changed in this way.